Source/WebCore/PAL/pal/mac/ScreenCaptureKitSoftLink.h

```diff
@@ -37,11 +37,9 @@ SOFT_LINK_CLASS_FOR_HEADER_WITH_AVAILABILITY(PAL, SCContentFilter, API_AVAILABLE
 SOFT_LINK_CLASS_FOR_HEADER_WITH_AVAILABILITY(PAL, SCStreamConfiguration, API_AVAILABLE(macos(12.3)))
 SOFT_LINK_CLASS_FOR_HEADER_WITH_AVAILABILITY(PAL, SCStream, API_AVAILABLE(macos(12.3)))
 SOFT_LINK_CLASS_FOR_HEADER(PAL, SCContentSharingSession)
+SOFT_LINK_CLASS_FOR_HEADER(PAL, SCContentSharingPicker)
```

Comment:
Probably not a big deal but should it be covered by HAVE_SC_CONTENT_SHARING_PICKER?